site/content/en/docs/Third Party Content/libraries-tools.md

```diff
@@ -31,3 +31,7 @@ Libraries or applications that implement messaging systems.
 
 - [agones-allocator-client](https://github.com/FairwindsOps/agones-allocator-client) - A client for testing allocation servers.
   Made by [Fairwinds](https://fairwinds.com)
+  
+## Development Tools
+
+= [Minikube Agones Cluster](https://github.com/comerford/minikube-agones-cluster) - Automates the creation of a complete Kubernetes/Agones Cluster locally, using Xonotic as a sample gameserver. Intended to provide a local environment for developers which approximates a production Agones deployment.
```

Comment:
Should be - instead of =. See https://ba5a993-dot-preview-dot-agones-images.appspot.com/site/docs/third-party-content/libraries-tools/